exercism/diamond/diamond.js

```diff
@@ -0,0 +1,12 @@
+var Diamond = function () {}
```

Comment:
this function doesnt work yet right? what feedback were you looking for originally?